### ISSUE TITLE
How to convert TimeSformer Implementation For Regression Tasks

### ISSUE THREAD

**Issue by Taimoor-R** [mx-mark/VideoTransformer-pytorch#30]
Hi, Great work! I was just wondering how to modify the implemented code for a regression task. Furthermore, can we use custom dataloader for the datasets as the dataset I am using is frames and thier corresponding values(regression labels).
